packages/client/components/NewMeetingCarousel.tsx

```diff
@@ -86,24 +87,54 @@ interface Props {
   idx: number
   setIdx: (idx: number) => void
   meetingOrder: MeetingTypeEnum[]
+  onStartMeetingClick: () => void
 }
 
 const NewMeetingCarousel = (props: Props) => {
-  const {idx, setIdx, meetingOrder} = props
+  const {idx, setIdx, meetingOrder, onStartMeetingClick} = props
 
   // TODO: remove when standups feature flag removed
   const moreThanThreeSlides = meetingOrder.length > 3
 
+  useHotkey('left', () => {
+    if (idx !== 0) {
+      const newIdx = idx - 1
+      setIdx(newIdx)
+    }
+  })
+  useHotkey('right', () => {
+    if (idx !== meetingOrder.length - 1) {
+      const newIdx = idx + 1
+      setIdx(newIdx)
+    }
+  })
+
+  const enterKey = 13
+  // keycode is a number but package thinks it's a string
+  const onKeyPress = (_swiper: unknown, keycode: string) => {
+    if (parseInt(keycode, 10) === enterKey) {
+      // inefficient, but only happens on enter
+      const isModalOpen =
+        document.querySelector(`div[id='templateModal']`) ||
```

Comment:
-1 i'd love to avoid resorting to checking the DOM & keep on the logic in the react lifecycle.

there's a few ways to do this, top of mind is to use focus:
- focus the meeting card with the active index (if it's a div you can use tabIndex={0} to make it focusable)
- put the keypress handler on the active card
- stop propagation if necessary

Reply:
Cool, TIL about tabIndex!